src/WebSdk/Publish/Tasks/Tasks/TransformWebConfig.cs

```diff
@@ -138,5 +144,23 @@ public override bool Execute()
             Log.LogMessage(MessageImportance.Low, "Configuring project completed successfully");
             return true;
         }
+
+        /// <summary>
+        /// Searches for an existing (case-insensitive) `Web.config` file. Otherwise defaults to defaultWebConfigName
+        /// </summary>
+        /// <param name="projectPath">Full path to project file</param>
+        /// <param name="defaultWebConfigName">Web Config file name to search for i.e. `web.config`</param> 
+        /// <returns></returns>
+        public string GetWebConfigFileOrDefault(string projectPath, string defaultWebConfigName)
+        {
+            var projectDirectory = Path.GetDirectoryName(projectPath);
+            var currentWebConfigFileName = Directory.EnumerateFiles(projectDirectory)
+                .FirstOrDefault(file => string.Equals(Path.GetFileName(file), defaultWebConfigName, StringComparison.CurrentCultureIgnoreCase)); 
```

Comment:
Thanks for making this change. it would be best to change this to [StringComparison.OrdinalIgnoreCase](https://learn.microsoft.com/en-us/dotnet/api/system.stringcomparison#system-stringcomparison-ordinalignorecase) instead.